docs/how-to/quick-test-stack.md

```diff
@@ -0,0 +1,59 @@
+---
+comments: true
+---
+
+A quick and easy method for testing a running CDK stack, whether in validium or rollup mode, is by sending a zero-value transaction and examining the result.
+
+## Send transaction with cast
+
+In a terminal window, run the following command where the mnemonic is used just for testing and the address can be any valid account address. 
+
+```sh
+cast send --legacy --mnemonic 'code code code code code code code code code code code quality' --value 0 --gas-price 0 --rpc-url http://127.0.0.1:8123 0x0bb7AA0b4FdC2D2862c088424260e99ed6299148
```

Comment:
You might need to explain how to get this ip address (like you did in the section below).

Reply:
done, reordered for clarity